frontend/console/src/features/infrastructure/RoutesList.tsx

```diff
@@ -1,3 +1,27 @@
-export const RoutesList = () => {
-  return <>Routes Content</>
+import { AttributeBadge } from '../../components'
+import { List } from '../../components/List'
+import type { StatusResponse_Route } from '../../protos/xyz/block/ftl/v1/ftl_pb'
+
+export const RoutesList = ({ routes }: { routes: StatusResponse_Route[] }) => {
+  return (
+    <List
+      items={routes}
+      renderItem={(route) => (
+        <div className='flex w-full'>
+          <div className='flex gap-x-4 items-center w-1/2'>
+            <div className='whitespace-nowrap'>
+              <div className='flex gap-x-2 items-center'>{route.module}</div>
+              <p className='mt-1 flex text-xs leading-5 text-gray-400 font-roboto-mono'>{route.endpoint}</p>
```

Comment:
nit: this shade of gray feels good for dark mode but a little eye-strainy for light mode. What do you think about bumping it up to 600?

Reply:
Bumped to `500` which feels pretty good too